db/builder.cc

```diff
@@ -61,7 +61,8 @@ TableBuilder* NewTableBuilder(
 Status BuildTable(
     const std::string& dbname, Env* env, const ImmutableCFOptions& ioptions,
     const MutableCFOptions& mutable_cf_options, const EnvOptions& env_options,
-    TableCache* table_cache, InternalIterator* iter, FileMetaData* meta,
+    TableCache* table_cache, InternalIterator* iter,
+    ScopedArenaIterator range_del_iter, FileMetaData* meta,
```

Comment:
Since ScopedArenaIterator has disabled copy constructor, maybe it is better to make it explicit with rvalue reference.